src/httpauth/digest.c

```diff
@@ -16,21 +16,37 @@
 #include <re_httpauth.h>
 
 
+enum {
+	CNONCE_NC_LENGTH = 9, /* 8 characters + '\0' */
```

Comment:
perhaps SIZE is better ?

if you know the length, why do you need a trailing null terminator ?